pkg/test/framework/components/prometheus/kube.go

```diff
@@ -115,7 +115,7 @@ func (c *kubeComponent) API() prometheusApiV1.API {
 func (c *kubeComponent) WaitForQuiesce(format string, args ...interface{}) (model.Value, error) {
 	var previous model.Value
 
-	time.Sleep(time.Second * 5)
+	time.Sleep(time.Second * 30)
```

Comment:
Can you add a TODO to fix this? We should eventually get rid of all of our sleeps.

Reply:
Testing it again without the increased sleep.. I did some refactoring to make things faster